static/app/components/avatar/actorAvatar.tsx

```diff
@@ -31,9 +31,9 @@ function LoadTeamAvatar({
   ...props
 }: {teamId: string} & Omit<React.ComponentProps<typeof TeamAvatar>, 'team'>) {
   const {teams, isLoading} = useTeamsById({ids: [teamId]});
-  const team = teams.find(t => t.id === teamId);
+  const team = teams.find(t => t.id === teamId) ?? null;
```

Comment:
Do you need the `?? null`. Won't it return `undefined`? is there a difference here

Reply:
https://github.com/getsentry/sentry/blob/28a38c7e5b412c53da4eae0d432575846992c16b/static/app/components/avatar/teamAvatar.tsx#L5-L7